__tests__/workers/personalizedDigestEmail.ts

```diff
@@ -9,13 +9,50 @@ import { sourcesFixture } from '../fixture/source';
 import { sendEmail } from '../../src/common';
 import nock from 'nock';
 import { subDays } from 'date-fns';
+import { features, getUserGrowthBookInstace } from '../../src/growthbook';
+import { sendAnalyticsEvent } from '../../src/integrations/analytics';
 
 jest.mock('../../src/common', () => ({
   ...(jest.requireActual('../../src/common') as Record<string, unknown>),
   sendEmail: jest.fn(),
   createEemailBatchId: jest.fn(),
 }));
 
+jest.mock('../../src/integrations/analytics', () => ({
+  ...(jest.requireActual('../../src/integrations/analytics') as Record<
+    string,
+    unknown
+  >),
+  sendAnalyticsEvent: jest.fn(),
+}));
+
+getUserGrowthBookInstace;
```

Comment:
What's this?

Reply:
No clue, most have been autocomplete artefact.